src/graphlog-comparison/sapling-curved.txt

```diff
@@ -0,0 +1,225 @@
+
+        @  node 1
+        
+
+        @  many
+        │  lines
+        │  of
+        │  text
+        o  single line
+        
+
+        @
+        │
+        │  many
+        │
+        │  lines
+        │
+        │  of
+        │
+        │  text
+        │
+        │
+        o  single line
+        
+
+        @  node 3
+        o  node 2
+        o  node 1
+        
+
+        o  node 7
+        │ o  node 6
+        o │  node 5
+        │ o  node 4
+        @ │  node 3
+        │ o  node 2
+        o  node 1
+        
+
+        o  node 3
+        │
+        ~
+
+        o  node 2
+        │
+        ~
+
+        @  node 1
+        │
+        ~
+        
+
+        o  node 4
+        │ o  node 3
+        o │  node 2
+        │ │
+        ~ │
+          │
+          o  node 1
+        
+
+        o  node 4
+        │ o  node 3
+        o │  node 2
+          │  more
+          │  text
+          o  node 1
+        
+
+        o  node 4
+        │ o  node 3
+        │ o  node 2
+        │ │
+        │ ~
+        │
+        o  node 1
+           more
+           text
+        
+
+        o  node 3
+        │ o  node 2
+        │ │  with
+        │ ~  long
+        │    description
+        o  node 1
+        
+
+        @  node 4
+        │ o  node 3
+        ├─╯
+        │ o  node 2
+        ├─╯
+        o  node 1
+        
+
+        o  node 10
+        │ o  node 9
+        │ │ o  node 8
+        o │ │  node 7
+        │ o │  node 6
+        │ │ o  node 5
+        o │ │  node 4
+        │ o │  node 3
+        ├─╯ │
+        │   o  node 2
+        ├───╯
+        o  node 1
+        
+
+        o  node 5
+        │ o  node 4
+        │ │ o  node 3
+        ├───╯
```

Comment:
I think I like it better if the horizontal line is broken, not the vertical one (the horizontal gap is shorter)...

Reply:
cc @markbt for consideration.